docs/jenkins-setup.txt

```diff
@@ -0,0 +1,161 @@
+=== Machine setup ===
+- Download Debian netinst, check sha512sum (and signature of sha512sum), use Startup Disk Creator (preinstalled on Ubuntu) to make flash drive
+- In BIOS (F2, and then F2 again later), set flash drive mode to hard disk and put flash drive to the top of boot sequence
+- Load firmware from http://cdimage.debian.org/cdimage/unofficial/non-free/firmware/stretch/current/, need bnx2 and qlogic
+	- Via usb:
+		lsblk
+		mkdir /media/<drive name> # <drive name> is entirely arbitrary
+		mount /dev/sdb1 /media/<drive name>
+		(remember to umount afterwards)
+- Network config:
+	- address: 18.4.60.253
+		(from `stella hijinks`)
+	- netmask: 255.255.254.0
+		(from `moira 5 1 sipb-460`)
+	- gateway: 18.4.60.1
+		(from `moira 5 1 sipb-460`, but note that
+			the last octet should be 1, by convention
+			for network gateways)
+	- nameservers: 18.72.0.3 18.70.0.160 18.71.0.151
+		(from `dig mit.edu` from within MIT's network)
+
+=== Standard setup ===
+apt update
+apt upgrade
+apt dist-upgrade
+apt install sudo openssh-server git
+apt install vim htop less xclip
```

Comment:
Why do we need xclip on a (presumably headless) server?

Reply:
It's used to share clipboards over SSH in vim.